admin/admin-service/src/main/java/io/mosip/admin/AdminBootApplication.java

```diff
@@ -8,7 +8,8 @@
 import org.springframework.scheduling.annotation.EnableAsync;
 import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
 
-@SpringBootApplication(scanBasePackages = { "io.mosip.admin.*", "${mosip.auth.adapter.impl.basepackage}" })
+@SpringBootApplication(scanBasePackages = { "io.mosip.admin.*", "io.mosip.commons.*", 
+		"${mosip.auth.adapter.impl.basepackage}","io.mosip.commons.packet.spi.*" })
```

Comment:
"io.mosip.commons.*" is already present, do we really require "io.mosip.commons.packet.spi.*" ?

Reply:
It is not required and removed as suggested